### PR TITLE
Tail Overhaul 2

### PR DESCRIPTION
**Pull request by Spookat**
Updated Geeves' original Tail Overhaul #12785 with fixes that caused the SQL to break.

Adds a short tail variant for Tajara, and cloth accessories that can be attached to regular tails. More sprites will come if it gets merged.

Playtested, everything appears to be working correctly. Tail storage can be accessed under the Abilities tab.

As this PR is quite old, please let me know if there's anything wonky that could be changed.

### CODE REVIEW

**Comment by Spookat** [comment]
!review

File: code/modules/organs/subtypes/standard.dm [hunk 1]
```diff
@@ -43,6 +43,21 @@
...
+
+/obj/item/organ/external/groin/Destroy()
+	QDEL_NULL(tail_storage)
+	return ..()
+
+/obj/item/organ/external/groin/removed()
+	. = ..()
+	for(var/thing in tail_storage)
+		tail_storage.remove_from_storage(thing, get_turf(src))
+
```
Comment (NonQueueingMatt) [inline code/modules/organs/subtypes/standard.dm:60]: It seems a bit suspect that this doesn't check if the species actually has a tail.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

File: SQL/migrate/V076__tail_overhaul.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+--
+-- Implemented in PR #numberhere.
```
Comment (SleepyGemmy) [inline SQL/migrate/V076__tail_overhaul.sql:2]:
```suggestion
-- Implemented in PR #15577.
```

File: code/game/objects/items/weapons/storage/internal.dm [hunk 3]
```diff
@@ -97,3 +97,48 @@
 /obj/item/storage/internal/skrell/Initialize()
 	. = ..()
 	name = initial(name)
+
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/internal.dm:100]:
```suggestion
```

File: code/modules/clothing/tail_accessory.dm [hunk 4]
```diff
@@ -0,0 +1,29 @@
+/obj/item/clothing/tail_accessory
+	name = "tail accessory"
+	desc = "If you're seeing this, something went wrong! Please report it on Github."
```
Comment (SleepyGemmy) [inline code/modules/clothing/tail_accessory.dm:3]:
```suggestion
	desc = DESC_PARENT
```

File: code/modules/mob/living/carbon/human/human_powers.dm [hunk 5]
```diff
@@ -1320,3 +1320,18 @@ mob/living/carbon/human/proc/change_monitor()
...
+	set name = "Tail Accessories"
+	set desc = "Opens the tail accessory slot."
+	set category = "Abilities"
+
+	var/obj/item/organ/external/groin/G = organs_by_name[BP_GROIN]
+	if(!G)
+		to_chat(usr, SPAN_WARNING("You have no tail!"))
+		return
+	if(!G.tail_storage)
+		to_chat(usr, SPAN_WARNING("Something went wrong with your tail storage, it's missing! (Report this on Github)"))
```
Comment (SleepyGemmy) [inline code/modules/mob/living/carbon/human/human_powers.dm:1334]:
```suggestion
		to_chat(usr, SPAN_WARNING("Your tail storage is missing!"))
```

**Pull request merged**